swagger_to_sdk_config.json

```diff
@@ -6,7 +6,7 @@
   },
   "projects": {
     "azure_mgmt_analysis_services": {
-      "swagger": "arm-analysisservices/2016-05-16/swagger/analysisservices.json",
+      "swagger": "specification/analysisservices/resource-manager/readme.md",
```

Comment:
New Api Version was always a problem, since we weren't able to create a PR for this case (since it's a new file). With MD, this problem is solving by itself. I would recommend not using a tag, anyway you get a PR, so you control what you merge.
Note that even if it's not related, you can use the "build_dir" key to build a metadata file about generation:
https://github.com/Azure/azure-sdk-for-python/blob/master/azure-mgmt-authorization/build.json